server/socket.js

```diff
@@ -63,40 +65,56 @@ export default function startSocketServer(http) {
     });
 
 
-    socket.on(CS.ADD_MESSAGE, data => {
-      Message.add(data, (err, result) => {
-        io.to(data.channelId).emit(SC.ADD_MESSAGE, result.toObject());
+    socket.on(CS.ADD_MESSAGE, ({senderId, channelId, text}) => {
+      if (!(
+        validate(senderId).isString().inRange(20, 30).end() &&
+        validate(channelId).isString().inRange(20, 30).end() &&
+        validate(text).isString().inRange(0, 1000).end()
+      )) return;
+      Message.add({ senderId, channelId, text }, (err, result) => {
```

Comment:
Same with `senderId`, what if there's no such sender. Will Mongoose validate it somehow and throw error? (that would be correct a response)